ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/jsonrpc/EeaSendTransactionJsonParameters.java

```diff
@@ -134,4 +141,10 @@ private void validateValue(final String value) {
           "Non-zero value, private transactions cannot transfer ether");
     }
   }
+
+  private static PrivacyIdentifier encodeStringToIso8559(final String input) {
+    return input.startsWith("0x")
```

Comment:
This looks like it creating the privacy identifier not encoding to iso8559

Reply:
Done.